assets/js/sections/mode.js

```diff
@@ -30,9 +31,9 @@ $('.modetable').DataTable({
 						select.append('<option value="' + d + '">' + d + '</option>');
 					});
 			});
+		table.columns.adjust().draw();
 	},
 });
-$($.fn.dataTable.tables(true)).DataTable().columns.adjust();
```

Comment:
<img width="465" alt="error" src="https://github.com/wavelog/wavelog/assets/80885850/4b159edb-b65a-4b0d-aae0-c801a04cca66">

This error appeared. Seems like a kind of race conditions problem, Nothing serious. `adjust().draw()` fixes it.